app/assets/stylesheets/preact/onboarding-modal.scss

```diff
@@ -36,7 +36,7 @@
   @media screen and (min-width: $breakpoint-s) {
     justify-content: center;
     padding: var(--su-7) var(--su-8);
-    height: 800px;
+    height: auto;
   }
 }
 
```

Comment:
Alongside suggestion from @aitchiss here's one more to prevent jumping things too much:

```
// Adjusting modals
.onboarding-main {
  --onboarding-modal-height: 800px;
  
  &.introduction {
    --onboarding-modal-height: unset;
  }
  
  .crayons-modal__box {
    height: var(--onboarding-modal-height);
  }
}
```

(explained in the Conversation comment)